README.md

```diff
@@ -40,6 +40,8 @@ The tourniquet should be packaged in clear, thin plastic such as PVC wrap to pre
 
 ## Injection molding - WARNING
 We have presented injection molding files in the [`injection_molding`](injection_molding) folder. The injection molded tourniquet has not yet been deployed and is in the process of being tested at this time. Follow best practices for injection molding and assemble parts as above.
+The "desktop_injection" folder is highly experimental and not deployed yet. We give no guarantees to the mold's correct behavior. 
```

Comment:
The "clip" mold is technically done, we need to do a final "boolean" operation on it. The model part from Aalbers is corrupted though, which makes it so we can't really realize that final step. We have a lead on alternative models from our FreeCAD designers, we'll also be trying to fix the Aalbers model to avoid losing on two days of CAD work.

Reply:
We've since iterated on the clip mold. We needed to redesign it twice and are on the third iteration, to be tested on Saturday.

Reply:
Clip is giving us problems due to the lack of rigidity of the material and the small jaw range of our vise. I got a promising result and opted for milling it out to avoid the shattering. We'll print out more resin molds of it as well in time,